app/bundle/src/main/scala/org/bitcoins/bundle/gui/BundleGUI.scala

```diff
@@ -43,6 +44,10 @@ object BundleGUI extends WalletGUI with JFXApp {
     val usedDir = DatadirUtil.getFinalDatadir(datadir, baseConfig, None)
 
     System.setProperty("bitcoins.log.location", usedDir.toAbsolutePath.toString)
+
+    if (Properties.isWin) {
+      System.setProperty("HOME", datadir.getParent.toAbsolutePath.toString)
```

Comment:
What is this? Things like this scare me. Why do we need a special exception on windows? At least add a comment